contracts/payloads/IGP110/PayloadIGP110.sol

```diff
@@ -0,0 +1,476 @@
+        IFluidReserveContract(RESERVE_CONTRACT_PROXY).revoke(protocols_, tokens_);
+    }
+
+    /// @notice Action 2: Revenue collection for buyback
+    function action2() internal isActionSkippable(2) {
+        {
+            address[] memory tokens = new address[](9);
+
+            tokens[0] = USDT_ADDRESS;
+            tokens[1] = wstETH_ADDRESS;
+            tokens[2] = ETH_ADDRESS;
+            tokens[3] = USDC_ADDRESS;
+            tokens[4] = sUSDe_ADDRESS;
+            tokens[5] = cbBTC_ADDRESS;
+            tokens[6] = WBTC_ADDRESS;
+            tokens[7] = GHO_ADDRESS;
+            tokens[8] = USDe_ADDRESS;
+
+            LIQUIDITY.collectRevenue(tokens);
+        }
+        {
+            address[] memory tokens = new address[](9);
+            uint256[] memory amounts = new uint256[](9);
+
+            tokens[0] = USDT_ADDRESS;
+            amounts[0] =
+                IERC20(USDT_ADDRESS).balanceOf(address(FLUID_RESERVE)) -
+                10;
+
+            tokens[1] = wstETH_ADDRESS;
+            amounts[1] =
+                IERC20(wstETH_ADDRESS).balanceOf(address(FLUID_RESERVE)) -
+                0.1 ether;
+
+            tokens[2] = ETH_ADDRESS;
+            amounts[2] = address(FLUID_RESERVE).balance - 0.1 ether; // 0.1 ETH
+
+            tokens[3] = USDC_ADDRESS;
+            amounts[3] =
+                IERC20(USDC_ADDRESS).balanceOf(address(FLUID_RESERVE)) -
+                10;
+
+            tokens[4] = sUSDe_ADDRESS;
+            amounts[4] =
+                IERC20(sUSDe_ADDRESS).balanceOf(address(FLUID_RESERVE)) -
+                0.1 ether;
+
+            tokens[5] = cbBTC_ADDRESS;
+            amounts[5] =
+                IERC20(cbBTC_ADDRESS).balanceOf(address(FLUID_RESERVE)) -
+                10;
+
+            tokens[6] = WBTC_ADDRESS;
+            amounts[6] =
+                IERC20(WBTC_ADDRESS).balanceOf(address(FLUID_RESERVE)) -
+                10;
+
+            tokens[7] = GHO_ADDRESS;
+            amounts[7] =
+                IERC20(GHO_ADDRESS).balanceOf(address(FLUID_RESERVE)) -
+                10;
+
+            tokens[8] = USDe_ADDRESS;
+            amounts[8] =
+                IERC20(USDe_ADDRESS).balanceOf(address(FLUID_RESERVE)) -
+                10;
+
+            FLUID_RESERVE.withdrawFunds(tokens, amounts, TEAM_MULTISIG);
+        }
+    }
+
+    /// @notice Action 3: Set dust limits for syrupUSDT DEX and vaults
+    function action3() internal isActionSkippable(3) {
+        {
+            // dust limits for SYRUPUSDT-USDT DEX (DEX ID 40)
+            address syrupUSDT_USDT_DEX = getDexAddress(40);
+            // SYRUPUSDT-USDT DEX
+            DexConfig memory DEX_syrupUSDT_USDT = DexConfig({
+                dex: syrupUSDT_USDT_DEX,
+                tokenA: syrupUSDT_ADDRESS,
+                tokenB: USDT_ADDRESS,
+                smartCollateral: true,
+                smartDebt: false,
+                baseWithdrawalLimitInUSD: 7_000_000, // $7M
```

Comment:
limits should be dust here for all

setDexAuth and setVaultAuths must be set to true